tests/integration/llm/prepare.py

```diff
@@ -81,7 +81,19 @@
     "t5-small": {
         "option.model_id": "t5-small",
         "option.tensor_parallel_degree": 4,
-    }
+    },
+    "gpt2-xl": {
+        "option.model_id": "gpt2-xl",
+        "option.tensor_parallel_degree": 1,
+    },
+    "facebook/opt-13b": {
+        "option.model_id": "facebook/opt-13b",
```

Comment:
This model is in our S3 bucket, using s3 bucket downloading will speed up

Reply:
updated